config/sync/og.og_role.rdf_entity-collection-author.yml

```diff
@@ -12,10 +12,20 @@ group_type: rdf_entity
 group_bundle: collection
 group_id: null
 permissions:
+  - 'transition own node discussion from deletion_request to validated using the community_content - node:pre_moderated workflow'
+  - 'transition own node discussion from draft to validated using the community_content - node:pre_moderated workflow'
+  - 'transition own node discussion from proposed to validated using the community_content - node:pre_moderated workflow'
+  - 'transition own node discussion from validated to validated using the community_content - node:pre_moderated workflow'
+  - 'transition own node document from deletion_request to validated using the community_content - node:pre_moderated workflow'
   - 'transition own node document from draft to validated using the community_content - node:pre_moderated workflow'
+  - 'transition own node document from proposed to validated using the community_content - node:pre_moderated workflow'
   - 'transition own node document from validated to validated using the community_content - node:pre_moderated workflow'
+  - 'transition own node event from deletion_request to validated using the community_content - node:pre_moderated workflow'
```

Comment:
The pre-moderated workflow works differently than the post moderated workflow as you know already. The fact is that since we are adding an extra workflow to the system, it is normal that we copy over the permissions from the rest of the workflows. The Functional team (discussed it) expects of course to have the same behavior as the rest of the community content. This permission exists in the rest of the CC so I simply put it there. The fact that the Workflow tests are also passing, it means that all permissions are identical.